internal/testrunner/runners/system/tester.go

```diff
@@ -2057,6 +2057,10 @@ func selectPolicyTemplateByName(policies []packages.PolicyTemplate, name string)
 }
 
 func (r *tester) checkTransforms(ctx context.Context, config *testConfig, pkgManifest *packages.PackageManifest, ds kibana.PackageDataStream, dataStream string, syntheticEnabled bool) error {
+	if config.SkipTransformValidation {
+		return nil
+	}
+
```

Comment:
Due to the errors found in the `elasticsearch` package from integrations (see PR description) when testing to check each index independently, it looks like it would be needed to add this kind of setting in the system tests.